src/docs/sphinx/tutorials/step02/Tutorial.rst

```diff
@@ -75,7 +75,7 @@ The differences are worth pointing out here. In GEOSX:
    vertices can be anywhere in space. This is useful for grids that turn, fold,
    or are heavily bent. Hexahedral blocks should nevertheless not be deprecated
    and have 8 distinct vertices.
-   Some tolerance exists for deprecation to wedges or prisms
+   Some tolerance exists for deprecation to wedges
```

Comment:
I think the verb we want is "degenerate" or maybe "collapse", not "deprecate."  Maybe check Hughes on this one.

Reply:
collapse is more accurate.

Reply:
After checking Hughes', I change to:

or are heavily bent. Hexahedral blocks should nevertheless **have 8 distinct
vertices that are not coalesced**.
Some tolerance exists for **degeneration** to wedges